streamer/src/nonblocking/quic.rs

```diff
@@ -1020,7 +1027,7 @@ pub mod test {
             MAX_STAKED_CONNECTIONS,
             MAX_UNSTAKED_CONNECTIONS,
             stats.clone(),
-            1000,
+            2000,
```

Comment:
What problems are we seeing to cause us to want to raise this timeout? Is it just some CI tests? I'm concerned that this may lead to greater potential for denial of service...

Reply:
Yes -- we primarily see this in our CI builds. Hard to reproduce locally but been seen sporadically seen by folks including myself. The change itself should not increase any denial of service as it still honor the timeout requirement.